### DISCUSSION TITLE
How to fix height portal?

### DISCUSSION THREAD

**Discussion by Zhanadil1509** [mantinedev/mantine#7159] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.14.1

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

I use Menu in Drawer

![image](https://github.com/user-attachments/assets/943df11b-4d3d-4d90-9452-207f9999fa00)

```
<Menu
                  portalProps={{
                    style: { height: "100% !important" },
                  }}
                  trigger="hover"
                  position="right-start"
                >
```
                i'm added portalProps, but not working
                
![image](https://github.com/user-attachments/assets/8333211a-db94-45c3-8137-94d3bed1811f)
but when i set in portal custom height, showing correctly. How can fix that?

### Self-service

- [X] I would be willing to implement a fix for this issue

## Update
i tried use vh or px, it's worked. But, if i have big menu.

![image](https://github.com/user-attachments/assets/24e30afa-a27c-43b2-887f-33512c459a67)

fix not good.
```
.menu {
  bottom: 0;
  overflow: auto;
  border: none;
  height: 88vh;
  &.mantine-Menu-dropdown {
    z-index: 999999;
    height: 100%;
  }
}
```
![image](https://github.com/user-attachments/assets/6f62fe53-d57e-42e9-97cc-2315f92445df)

What is the best way to fix it?

**Comment by rtivital** [comment]
Height 100% does not usually work, see https://stackoverflow.com/questions/7049875/why-doesnt-height-100-work-to-expand-divs-to-the-screen-height

Use other units to assign size, for example `vh`.
